### PR TITLE
Add Rlisa33.md

### PR DESCRIPTION
**Pull request by Rlisa33**
# **Enjoying Github Intern Software Delevopment Service!**
[Rachel]  Mathurin (https://github.com/Rlisa33/Rlisa33.github.io)
win_20170328_20_18_28_pro

GitHub Intern!
 Rachel Mathurin
 Eastern Time Florida
 OS Windows 7
 I am an online student at the University of Phoenix. I am studying to earn my Master of Science in Accountancy. 
 I lived in Florida for over four years now. I like reading, and surfing the net. I like drawing. 
 I like software engineering and computer science and math as well as accounting.
 I am eager to get a job to start working and I believe volunteering with help my resume.

### CODE REVIEW

**Comment by waryhermit** [comment]
Hooray! you made a pull request. Now you need to add at least three mark down elements. I suggest link, picture, and header as good ones to try.

**Comment by Supriyaranjan** [comment]
Good one rachel !

**Comment by waryhermit** [comment]
Good changes, but you need to make those changes to the file itself, not the pull request description. One way to do that is at the top of this pull request where it has  `Conversation 2 Commits 3 Files changed 1 ` go to the files changed tab and then click the pencil icon.

**Comment by sarvanivadali** [comment]
@Rlisa33 Agreed. Please add few more markdown elements and you should get an approval then.

**Comment by paulbert** [comment]
Hi @Rlisa33 it looks like you need to move the file to the `/pages` directory and add an `.md` file extension.  Below is a link to a good tutorial on moving a file in GitHub and also make sure you put `.md` at the end of the file as well.  You can do this when you add in the markdown elements.

https://help.github.com/articles/moving-a-file-to-a-new-location/

File: Rlisa33.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#**Enjoying Github Intern Software Delevopment Service!**
```
Comment (waryhermit) [inline Rlisa33.md]: You need a space between the `#` and the `**`

File: Rlisa33.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+(https://github.com/Rlisa33/Rlisa33.github.io)
```
Comment (waryhermit) [inline Rlisa33.md]: Looks like you meant to make a list. To make a list, hit enter (make a new line) and then for each list item write `-` followed by a space and whatever text you want to be for that list item.

Comment (waryhermit) [inline Rlisa33.md]: You should add text to the link and make it a proper mark down element. To make a link you write it like this `[the text you want to turn into a link](http://actualLink.com)`

File: Rlisa33.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+win_20170328_20_18_28_pro
+GitHub Intern Experience!
```
Comment (waryhermit) [inline Rlisa33.md]: Looks like you meant to make a list. To make a list, add a new line between the list and other text, and then make list items by typing `-` and added a space after. Then you can write whatever you want on that list item.

File: Rlisa33.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# **Enjoying Github Intern Software Delevopment Service!**
```
Comment (waryhermit) [inline Rlisa33.md]: This should probably be your name

File: Rlisa33.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+[Rlisa33 Homepage](https://github.com/Rlisa33/Rlisa33.github.io)
+- (win_20170328_20_18_28_pro)
```
Comment (waryhermit) [inline Rlisa33.md]: Windows 10 Pro is sufficient -  the actual patch number will change all the time.

File: Rlisa33.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+- GitHub Intern Experience!
+- Rachel Mathurin
```
Comment (waryhermit) [inline Rlisa33.md]: Like I said before - move your name to the header

File: Rlisa33.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+- Eastern Time Florida
+- OS Window 7
```
Comment (waryhermit) [inline Rlisa33.md]: If you run multiple operating systems, list them next to each other.

File: Rlisa33.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
+# **Enjoying Github Intern Software Delevopment Service!**
+[Rlisa33 Homepage](https://github.com/Rlisa33/Rlisa33.github.io)
+- (win_20170328_20_18_28_pro)
+- GitHub Intern Experience!
+- Rachel Mathurin
+- Eastern Time Florida
```
Comment (waryhermit) [inline Rlisa33.md]: Location and time zone should be formatted more clearly something like these:

Location: Florida, USA
Time Zone: Eastern Standard Time (EST)

**OR**
Florida, USA (Eastern Standard Time)

File: Rlisa33.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+- OS Window 7
+- I am an online student at the University of Phoenix. I am studying to earn my Master of Science in Accountancy. 
+- I lived in Florida for over four years now. I like reading, and surfing the net. I like drawing. 
+- I like software engineering and computer science and math as well as accounting.
+- I am eager to get a job to start working and I believe volunteering with help my resume.
```
Comment (waryhermit) [inline Rlisa33.md]: Your bio should be a well formatted paragraph **or** you should make it read more like a list such as:

**Some facts about me:**

- I am an online student at the University of Phoenix studying to earn my Master of Science in Accountancy.
- I lived in Florida for over four years now.
 . . . and so on

File: Rlisa33.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+ ###I have all the thing required to do this job.
```
Comment (waryhermit) [inline Rlisa33.md]: You need to add a new line before this line and a space after the `###`

File: Rlisa33.md [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+ * A computer
+    * Internet
+    * Software of both Windows 7 and 10
+    * Processor RAM Memory 
+ * Skills 
+    * took previous courses in Software Developments and IOS
+    * Worked at a horticultural lab collecting data and processing it into the computer
+- _I want to be a success and you?_
```
Comment (waryhermit) [inline Rlisa33.md]: Seems a little out of place here - could you think of a way of incorporating it into your bio maybe?

File: Rlisa33.md (hunk 6 above)
Comment (Rlisa33) [inline Rlisa33.md]: This (win_20170328_20_18_28_pro) is a picture I copied to be part of the file.

**Pull request merged**